src/qibojit/backends/clifford_operations_gpu.py

```diff
@@ -0,0 +1,726 @@
+from functools import cache
+
+import cupy as cp  # pylint: disable=E0401
+import numpy as np
+from scipy import sparse
+
+name = "cupy"
+
+np = cp
```

Comment:
Here, `name` is never used, and `numpy` is imported just to be overwritten by `cupy`